drv/stm32h7-fmc-demo-server/src/main.rs

```diff
@@ -149,7 +167,7 @@ fn main() -> ! {
     const DATLAT: u8 = 0;
     // FMC_CLK division ratio relative to input (AHB3) clock, minus 1. Range:
     // 1..=15.
-    const CLKDIV: u8 = 15; // /4, for 50 MHz
+    const CLKDIV: u8 = 3; // /4, for 50 MHz (field is divisor-minus-one)
```

Comment:
Earlier in this set of commits, we tried to configure a 66.66 MHz bus speed.  What speed to we want to run at?

Reply:
We tried but did not succeed in configuring a 66 MHz bus speed, since the divisor is value-minus-one. It looks like we've only tested it at 50, so I wanted to get this merged at 50 before we play with accelerating it.